__tests__/numbers.test.js

```diff
@@ -77,18 +77,19 @@ describe('/numbers', () => {
   });
 
   describe('POST /multiply', () => {
-    xit('multiplies two numbers', done => {
+    it('multiplies two numbers', done => {
       request(app)
         .post('/numbers/multiply')
         .send({ a: 10, b: 3 })
         .then(res => {
+          console.log('Response received!');
```

Comment:
Should always remove console statements, especially debugging type ones like this